src/pydicom/pixels/encoders/__init__.py

```diff
@@ -0,0 +1 @@
+
```

Comment:
is this a line added?

Reply:
I think it just appears that way. Looking at https://github.com/pydicom/pydicom/blob/7a055cd556a87db8be93329acb3275ee4ccf21b9/src/pydicom/pixels/encoders/__init__.py it's empty.

Reply:
Your link shows `0 lines (0 loc) · 1 Byte`, so I guess there is a line feed there - but it doesn't really matter either way, was just curious.